web_widget_html_markdown/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2014 Sudokeys (<http://www.sudokeys.com>)
+# Copyright (C) 2017 Komit (<http://www.komit-consulting.com>)
+#
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Web Widget Html Markdown",
+    "version": "13.0.1.0.0",
+    "author": "Alexandre Díaz, "
+    "Komit, "
+    "Sudokeys, "
+    "Sunflower IT, "
+    "Odoo Community Association (OCA)",
+    "category": "Web",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/web",
+    "summary": "Widget to text fields that adds markdown Html bidirectional editor",
+    "depends": [
+        "web", "web_editor", 
+        ],
```

Comment:
Since we're not depending on `web_widget_text_markdown` anymore, we might as well remove that module from this PR, to clean it up a little.

Reply:
(Some other people picked up a standard migration for it here: https://github.com/OCA/web/pull/2136/commits)